GKE-On-Prem/guestbook.yaml

```diff
@@ -174,12 +174,17 @@ spec:
     tier: backend
     role: master
 ---
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: redis-master
 spec:
   replicas: 1
+  selector:
+    matchLabels:
+      app: redis
+      role: master
```

Comment:
```suggestion
      role: primary
```

Reply:
Good catch ❤️ 
Can you also suggest a change for the corresponding labels at the template level (Otherwise selector won't match)?.

Reply:
ah, only on diff lines you can add suggestions...so it should be done manually

Reply:
I've fixed manually. I couldn't replace any occurence of `master` and `slave` as some of those appear in Google's docs and should be fixed there first.